bundles/org.openhab.binding.teslascope/README.md

```diff
@@ -0,0 +1,241 @@
+# Teslascope Binding
+
+This binding integrates [Tesla Electrical Vehicles](https://www.tesla.com).
+
+The integration happens through the [Teslascope](https://www.teslascope.com) API.
+
+## Supported Things
+
+All Tesla vehicles supported by Teslascope are supported..
+
+## Discovery
+
+Auto-discovery is not currently supported.
+
+## Thing Configuration
+
+As a minimum, the IP address is needed:
```

Comment:
The IP address is not part if the following three liones of possible configuration settings.

Personally i also prefer a table for this configuration, including columns for default value, required and or advanced.

Reply:
Copy/paste error - I'll try and reformat, but won't be today.

Reply:
I completed the creation of a table.